core/services/keeper/validate.go

```diff
@@ -41,44 +41,39 @@ perform_upkeep_tx        [type=ethtx
 encode_check_upkeep_tx -> check_upkeep_tx -> decode_check_upkeep_tx -> encode_perform_upkeep_tx -> perform_upkeep_tx`
 )
 
-// expectedPipeline it is basically parsed expectedObservationSourceRaw value
-var expectedPipeline pipeline.Pipeline
-
-func init() {
-	pp, err := pipeline.Parse(expectedObservationSourceRaw)
+func ValidatedKeeperSpec() func(tomlString string) (job.Job, error) {
+	expectedPipeline, err := pipeline.Parse(expectedObservationSourceRaw)
```

Comment:
`pipeline.Parse` depends on cgo which is incompatible with fuzzing, so it can't be called from an `init()` in a fuzz test dependency. But there may be a better work around than this.

Reply:
This in itself is a worry, why does it depend on cgo?

Reply:
Gonum

Reply:
Actually, this is affecting using the 1.18 beta generally (not just fuzzing), so it may just be a matter of waiting for gonum to cut a release after go1.18 comes out: https://github.com/gonum/gonum#supported-go-versions

Reply:
Confirmed latest gonum fixes the issue. No official release cut yet though.